code/API_definitions/simple-edge-discovery.yaml

```
@@ -2,7 +2,7 @@
 openapi: 3.0.3
 info:
  title: Simple Edge Discovery API
```

According to guidelines `# title without "API" in it, e.g. "Number Verification"`
```suggestion
 title: Simple Edge Discovery
```

---

Thanks @rartych , committed